src/components/Navigation/SearchBar/SearchBar.vue

```diff
@@ -79,10 +79,11 @@ export default defineComponent({
     const filterVal: ComputedRef<string> = computed(
       () => store.state.nav.filterFoodParam
     );
+    // const filterVal = ref<any>("");
     const orderVal: ComputedRef<OrderOption> = computed(
       () => store.state.nav.foodListOrder
     );
-    const orderOptions: OrderOption[] = orderOpt.filter(option => option.type !== "none");
+    const orderOpts: OrderOption[] = orderOptions.filter(option => option.type !== "none");
```

Comment:
try not to shortening variable names, `orderOptions` was ok,
missing type for `option` param